js/search.js

```diff
@@ -968,6 +968,7 @@
 						} else */ if (learnsetEntry.indexOf(gen) < 0) {
 							continue;
 						}
+						if (this.gen === 8 && BattleMovedex[moveid].isNonstandard === "Past") continue;
```

Comment:
`if (learnsetEntry.indexOf(gen) < 0) {` should cover this...

Reply:
It doesnt

Reply:
Oh, of course it doesn't.

Reply:
My mistake.

Reply:
Well, we'll need to add an exception for NatDex formats.

Can we do that now, actually? Will make it easier later.